Lec/clann_morna.trig

```diff
@@ -406,7 +411,7 @@ _:missing-d23e02b6
 # to be gained from trying to tease apart two characters from this. EPT
 
 <#Forco>
-        a foaf:Person ;
+        a irishTitles:Rí ;
```

Comment:
I think we might have diverged in how we use `irishTitles:Rí`. I've been using it only in the context of regnal lists, where there is also succession data, whereas you are using it for everyone who is stated to be a king, even in isolation. Your use is probably better, but it does mean there will be quite a few kings scattered about only identified by `foaf:title`...